pages/common/acme.sh-dns.md

```diff
@@ -0,0 +1,25 @@
+# acme.sh --dns
+
+> Use a DNS-01 challenge to issue a certificate.
+> DNS API mode documentation: <https://github.com/acmesh-official/acme.sh/wiki/dnsapi>.
+> More information: <https://github.com/acmesh-official/acme.sh/wiki/DNS-manual-mode>.
+
+- Issue a certificate using an automatic DNS API mode:
+
+`acme.sh --issue --dns {{gnd_gd}} --domain {{example.com}}`
+
+- Issue a wildcard certificate (denoted by an asterisk) using an automatic DNS API mode:
+
+`acme.sh --issue --dns {{dns_namesilo}} --domain {{*.example.com}}`
+
+- Issue a certificate using a DNS alias mode:
+
+`acme.sh --issue --dns {{dns_cf}} --domain {{example.com}} --challenge-alias {{alias-for-example-validation.com}}`
+
+- Issue a certificate while disabling an automatic Cloudflare or Google DNS polling after the DNS record is added by specifying a manual wait time:
```

Comment:
Perhaps this will read better?
```suggestion
- Issue a certificate while disabling polling after the DNS record is added by specifying a custom wait time in seconds:
```

Reply:
It does specifically poll mentioned service providers. Some users consider this behavior a privacy concern. Without mentioning the actual names, it is not really clear what is the purpose. 

What about mentioning the word "privacy" at the very least?

Reply:
Oh, I see. What about this then?
```suggestion
- Issue a certificate while disabling automatic Cloudflare / Google DNS polling after the DNS record is added by specifying a custom wait time in seconds:
```